members/C001056.yaml

```diff
@@ -1,102 +1,103 @@
-          Foreign Affairs: Foreign Affairs
-          Healthcare: Healthcare
-          Housing: Housing
-          Immigration: Immigration
-          Interior: Interior
-          Judiciary: Judiciary
-          Labor: Labor
-          Law Enforcement: Law Enforcement
-          Medicare: Medicare
-          Religion: Religion
-          Service Academies: Service Academies
-          Small Business: Small Business
-          Social Security: Social Security
-          Taxes/Economy: Taxes/Economy
-          Telecommunications: Telecommunications
-          Trade: Trade
-          Transportation: Transportation
-          Veterans: Veterans
-          Welfare: Welfare
-      - name: State
-        selector: "#edit-submitted-address-administrative-area"
-        value: Texas
-        required: true
-    - find:
-      - selector: "#edit-submitted-e-newsletter-opt-in-1"
+        - name: Topic
+          selector: "#form-field-field_10b4eab"
+          value: $TOPIC
+          required: true
+          options:
+            Abortion: Abortion
+            Agriculture: Agriculture
+            Arts and Humanities: Arts and Humanities
+            Banking: Banking
+            Budget: Budget
+            Criminal Justice: Criminal Justice
+            Defense: Defense
+            Education: Education
+            Election Reform: Election Reform
+            Energy: Energy
+            Environment: Environment
+            FEMA: FEMA
+            Finance: Finance
+            Firearms: Firearms
+            Foreign Affairs: Foreign Affairs
+            Healthcare: Healthcare
+            Housing: Housing
+            Immigration: Immigration
+            Interior: Interior
+            Judiciary: Judiciary
+            Labor: Labor
+            Law Enforcement: Law Enforcement
+            Medicare: Medicare
+            Religion: Religion
+            Service Academies: Service Academies
+            Small Business: Small Business
+            Social Security: Social Security
+            Taxes/Economy: Taxes/Economy
+            Telecommunications: Telecommunications
+            Trade: Trade
+            Transportation: Transportation
+            Veterans: Veterans
+            Welfare: Welfare
+        - name: State
+          selector: "#form-field-field_562fb59"
+          options:
+            TX: TX
+          required: true
     - click_on:
-      - name: Newsletter opt out 
-        selector: "#edit-submitted-e-newsletter-opt-in-1"
-      - name: Submit
-        selector: input[type='submit'][value='Submit'] 
-    - find:
-      - selector: webform-submit button-primary btn btn-default signup form-submit
-    - find:
-      - selector: webform-submit button-primary btn btn-default signup form-submit
+        - name: Submit
+          selector: span.elementor-button-text
   success:
     body:
-      contains: Your message to Senator Cornyn's office has been submitted
-      
\ No newline at end of file
+      contains: Services
```

Comment:
Does it matter that the text "Services" appears on the page even before you submit the form?